Update configure-gcp.adoc

omri86 opened the pull request:
added some details on shared VPC and some minor fixes

@rcaballeromx - please see if you can add a link to the GCP permissions page

omri86 committed:
Update configure-gcp.adoc

added some details on shared VPC and some minor fixes

@rcaballeromx - please see if you can add a link to the GCP permissions page

rcaballeromx commented on compute/admin_guide/agentless-scanning/onboard-accounts/configure-gcp.adoc:
```diff
@@ -121,10 +121,10 @@ You can use wildcards to specify a range of tags in both keys and values followi
 If you are configuring a hub account, this requirement only applies to the hub account and not for the targets.
 
-... *Shared VPC*: If you are using a shared VPC, enter the shared VPC path in this field with the following convention. Go to the GCP documentation to https://cloud.google.com/vpc/docs/shared-vpc[learn more about shared VPCs]. Replace `{host_project_name}` with the ID of the project that owns the shared VPC.
+... *Shared VPC*: If you are using a shared VPC, enter the shared VPC path in this field with the following convention. Go to the GCP documentation to https://cloud.google.com/vpc/docs/shared-vpc[learn more about shared VPCs]. Replace `{host_project_name}` with the ID of the project that owns the shared VPC. Note that using a shared VPC also requires additional permissions set on the shared VPC h ...
```
Added the link to the permissions here.

Commit:
Add link. Grammar and style edits.

Pubs-MV committed:
Update configure-gcp.adoc

fixed style for "See" and "Refer" instead of "Go to".

and made links inline for more streamlined reading.

omri86 reviewed (commented):
LGTM
